git_theta/metadata.py

```diff
@@ -127,8 +127,7 @@ def from_commit(cls, repo: git.Repo, path: str, commit_hash: str) -> Metadata:
 
     @file_or_name(file="w")
     def write(self, file: TextIO):
-        metadata_dict = self.serialize()
-        json.dump(metadata_dict, file, indent=4, cls=MetadataEncoder)
+        json.dump(str(self), file, indent=4, cls=MetadataEncoder)
```

Comment:
I don't think you need to call json.dump again? Just need to do `file.write`?

Reply:
Thank you for your valuable feedback.Yes it should be ```file.write(str(self))```.